README.md

```diff
@@ -69,7 +69,7 @@ Prerequisites:
 
  * Make sure to have latest CMake installed. Minimum required version is 3.2
  * Compiler should support C++11 features. We have tested this SDK with gcc 5+, clang 3.8 and on Visual Studio 2015.
- * Openssl has version 1.0.2 and libssl-dev has version 1.0.2. OpenSSL v1.1.0 reference wrapper implementation is not included in this version of the SDK.
+ * Openssl has version 1.1.0 and libssl-dev has version 1.0.2. OpenSSL v1.1.0 reference wrapper implementation is not included in this version of the SDK.
```

Comment:
This is concerning.

Reply:
I fixed the issue with libssl-dev, it turns out there is a version `1.1`+, I just missed it in my initial Google search.

I left the part about "OpenSSL v1.1.0 reference wrapper implementation is not included in this version of the SDK" because I wasn't quite sure what it referred to. I can remove it as well though, I just wasn't sure if it's still accurate or not.